use_case_examples/federated_learning/README.md

```diff
@@ -0,0 +1,19 @@
+# Federated Learning
+
+This is an example on how to use a federated learning trained model with Concrete ML.
+This example is based on the example from the [flower blog post on LogisticRegression training using federated learning with flower](https://flower.dev/blog/2021-07-21-federated-scikit-learn-using-flower/).
+[Federated learning files source.](https://github.com/adap/flower/tree/9ee473152f2fde2a5b05b99829db27a607dc46ec/examples/sklearn-logreg-mnist)
+
+## Setup
+
+To run this example you will need to have `concrete-ml` installed and also install the extra requirements from `requirements.txt`, using `python -m pip install -r requirements.txt`.
+If you are using macOS with arm then you might need to launch the `./fix_grpc.sh` script for fix the `grpcio` package installation.
+
+## Train the model
+
+To train the model just use the `run.sh` script from the blogpost.
```

Comment:
From the blogpost? It's in this directory, no?

Reply:
Yes in this directory but the script itself was copied from the files of the blogpost.
I see how that can be confusing, I'll change that.